taurus/json/__init__.py

```diff
@@ -19,7 +37,7 @@ def loads(json_str, **kwargs):
     -------
     DictSerializable or List[DictSerializable]
         Deserialized versions of the objects represented by `json_str`, with links turned
-        back into pointers.
+        back into object references (python's pointers).
```

Comment:
```suggestion
        back into python object references.
```

Reply:
Python implements references with pointers under the hood, but the word pointer suggests a need for explicit dereference, which is the connotation I was trying to avoid.